### ISSUE TITLE
Bufferline shows buffer line when Alpha is open

### ISSUE THREAD

**Issue by LandonSchropp** [AstroNvim/astrocommunity#532]
### Checklist

- [X] I have searched through the AstroNvim docs
- [X] I have searched through the existing issues of AstroNvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.9.1

### Operating system/version

macOS 11.5

### Terminal/GUI

ITerm 2

### Describe the bug

When using the community config for Bufferline.nvim, the buffer is displayed when Neovim first opens. Instead, the bufferline should be hidden when Alpha is opened. (This is the behavior for the default Heirline configuration.)

### Steps to Reproduce

1. Set up a clean install of AstroNvim.
2. `mkdir -p ~/.config/nvim/lua/user`
3. Add the minimal repro config.

   ``` bash
   echo 'return {
     plugins = {
       { "AstroNvim/astrocommunity" },
       { import = "astrocommunity.bars-and-lines.bufferline-nvim" },
     },
   }' > ~/.config/nvim/lua/user/init.lua
   ```
6. Open/Close Neovim to install plugins.
7. Open Neovim.

### Expected behavior

When the editor opens and displays Alpha, the buffer should be hidden (`showbufferline` should be `0`)

### Screenshots

When the editor first opens:

<img width="1640" alt="Screen Shot 2023-08-22 at 2 59 00 PM" src="https://github.com/AstroNvim/astrocommunity/assets/361591/b20143aa-e686-4cb4-8904-5436a9169f83">

When running `:Alpha`:

<img width="1640" alt="Screen Shot 2023-08-22 at 2 58 54 PM" src="https://github.com/AstroNvim/astrocommunity/assets/361591/f43236ed-456f-468a-a5d4-cb2b9fee4a6d">

**Comment by Uzaaft** [comment]
https://www.reddit.com/r/neovim/comments/xyzifj/how_to_use_nvim_create_audocmd_to_create_autocmd/ should show you how to do this. @LandonSchropp  We'd appreciate it if you open a PR with this. :)

**Comment by LandonSchropp** [comment]
Thanks for the reply @Uzaaft!

I started down that road but got a little confused when reading through the Bufferline source code.

Take a look at these two functions:

* [toggle_bufferline](https://github.com/akinsho/bufferline.nvim/blob/417b303328118b6d836ae330142e88771c48a8a3/lua/bufferline.lua#L89-L93)
* [setup](https://github.com/akinsho/bufferline.nvim/blob/417b303328118b6d836ae330142e88771c48a8a3/lua/bufferline.lua#L201)

The way I interpret this is when setup is called, Bufferline should automatically hide the buffer if the number of buffers is less than 2 (which it should be when Alpha opens). I also noticed that Bufferline [already creates an autocmd for `BufAdd`](https://github.com/akinsho/bufferline.nvim/blob/417b303328118b6d836ae330142e88771c48a8a3/lua/bufferline.lua#L122-L129), so I was hesitant to add another that might conflict.

This problem seems to be specific to AstroNvim—I didn't notice the same issue with LazyVim, and looking through the LazyVim config, I didn't see any major differences in the Bufferline config.

Any thoughts on the best way to approach this?
